core/src/epicli/data/common/defaults/configuration/postgres-exporter.yml

```diff
@@ -0,0 +1,19 @@
+kind: configuration/postgres-exporter
+title: Postgres exporter
+name: default
+specification:
+  version: 0.8.0
+  file_name: postgres_exporter_v0.8.0_linux-amd64.tar.gz
```

Comment:
Do we allow user to specify version or file name, as we don't have them in repository? I think this should be in roles defaults and version should be used to generate a file name.

Reply:
It will not change much. You still need to put filename into requiremets.txt (see other components we use). Generate filename by joining name and version from variables will give additional work for user. Also we are not sure if next version's filename schema will change or not.

Reply:
If it changes, we anyway need to change version manually and check that it works, but most probably It will not require any other changes.

With such a configuration we can change file_name parameter and break the installation. In my opinion `specification` part should be used only for configurable settings.

Reply:
BTW, I also don't see where `version` value is used.

Reply:
Yes, you are right - in default we should keep configurable settings. But i think we should keep to one schema - like you can see for other exporters. I'm not saying that it's good practice, but this config fits to our (maybe bad) standards, and i'm not sure how in this case this could be done more clearly and easier to use. Version variable is not used in code, it's just variable for some future tasks like upgrade etc.

Reply:
IMO we should start changing it. And later do a task to refactor all roles that we have. What we have now is confusing to evenrybody :-1: :(

Reply:
I moved version and service description into role's defaults. And agree @sk4zuzu that we should refactor all roles with that case.